ome_zarr/reader.py

```diff
@@ -8,7 +8,11 @@
 import dask.array as da
 import numpy as np
 from dask import delayed
+from jsonschema import validate as jsonschema_validate
+from jsonschema.validators import validator_for
+from ngff.schemas import LocalRefResolver, get_schema
```

Comment:
As mentioned earlier today, if we go for an upstream packages containing the schemas as a resource, my vote goes towards prefixing the package/module with `ome` i.e. `ome_ngff.schemas`